src/Quartz.Extensions.DependencyInjection/JobFactoryOptions.cs

```diff
@@ -2,11 +2,6 @@ namespace Quartz
 {
     public class JobFactoryOptions
     {
-        /// <summary>
-        /// When DI has not been configured with the job type, should the default no-arg public constructor be tried.
-        /// </summary>
-        public bool AllowDefaultConstructor { get; set; }
```

Comment:
I think for 3.3 release it would be nicer to mark this as obsolete and and comment that no longer used, maybe link to GH issue?

Reply:
Added Obsolete attribute.